### ISSUE TITLE
Wrong SQL generated for INSERT .. SELECT .. ON DUPLICATE KEY IGNORE emulation using NOT EXISTS

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10989]
As reported here:

- https://groups.google.com/g/jooq-user/c/Gv-QnXU8uGE

This code:

```java
create().insertInto(TAuthor(), TAuthor_ID(), TAuthor_LAST_NAME())
        .select(select(val(3), val("Koontz")))
        .onDuplicateKeyIgnore()
        .execute();
```

Produces wrong SQL in Informix (if we turn off `MERGE` emulation, see #10990):

```sql
insert into "informix"."t_author" (
  "id",
  "last_name"
)
select
  "t"."v0",
  "t"."v1"
from (
  select
    null "v0",
    null "v1"
  from (
    select 1 as dual
    from systables
    where (tabid = 1)
  ) as dual
  where not (exists (
    select 1 "one"
    from "informix"."t_author"
    where (
      "informix"."t_author"."id" = null
      or "informix"."t_author"."last_name" = null
    )
  ))
) "t"
```

**Comment by lukaseder** [comment]
This can still be reproduced in Sybase ASE and VERTICA, which continue to emulate `ON DUPLICATE KEY IGNORE` using `NOT EXISTS`

**Comment by lukaseder** [comment]
This has simply nto been implemented. Only `INSERT .. VALUES .. ON DUPLICATE KEY IGNORE` is currently being emulated with the `INSERT .. SELECT .. NOT EXISTS` approach.

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.15.0 and 3.14.4 (#10991)
